lightning_pose/data/dali.py

```diff
@@ -375,11 +372,10 @@ def _setup_pipe_dict(
             "step": context_pred_cfg["sequence_length"] - 4,
             "batch_size": 1,
             "num_threads": self.num_threads,
-            "device_id": 0,
-            "random_shuffle": False,
```

Comment:
why has `random_shuffle` been removed? we definitely want this to be False when running inference

Reply:
ack, it's False by default but adding it back to be consistent